websites/B/Booyah/presence.ts

```diff
@@ -0,0 +1,215 @@
+let elapsed = Math.floor(Date.now() / 1000),
+	oldLang = "en",
+	prevUrl = document.location.href;
+
+const presence = new Presence({
+		clientId: "943158612090568745"
+	}),
+	getElement = (query: string): string | undefined => {
+		return document.querySelector(query)?.textContent;
+	},
+	getStrings = async () => {
+		return presence.getStrings(
+			{
+				play: "general.playing",
+				pause: "general.paused",
+				live: "general.live",
+				viewHome: "general.viewHome",
+				browse: "general.browsing",
+				watchingLive: "general.watchingLive",
+				watchingVid: "general.watchingVid",
+				viewCategory: "general.viewCategory",
+				viewTheir: "twitch.viewTheir",
+				channelSettings: "twitch.channelSettings",
+				dashboard: "twitch.dashboard",
+				viewFollow: "twitch.viewFollow",
```

Comment:
use twitch strings will need you to update the language setting accordingly. https://docs.premid.app/en/dev/presence/metadata#multilanguage

Reply:
I added twitch in multilanguage.